content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+
+```{r, message=FALSE, warning=FALSE}
+start_day = "2020-05-01"
+end_day = "2020-08-01"
+
+data_unadjusted <- suppressMessages(covidcast_signal("doctor-visits", "smoothed_cli", start_day = start_day,
+                                                     end_day = end_day,geo_type="state"))
+data_adjusted <- suppressMessages(covidcast_signal("doctor-visits", "smoothed_adj_cli", start_day = start_day, 
+                                                   end_day = end_day,geo_type="state"))
+
+data_unadjusted = data_unadjusted %>%
+  select(geo_value, signal, time_value, value) %>%
+  tidyr::pivot_wider(names_from=signal,values_from = value)
+
+data_adjusted = data_adjusted %>%
+  select(geo_value, signal, time_value, value) %>%
+  tidyr::pivot_wider(names_from=signal,values_from = value)
+
+cmb_df = inner_join(data_unadjusted,data_adjusted)
+
+cmb_df$geo_value = str_to_upper(cmb_df$geo_value)
+states_to_plot = c("AZ","CA","GA","FL","NJ","NY","PA","TX","WI")
+ggplot(cmb_df %>% filter(geo_value %in% states_to_plot)) +
+  geom_line(aes(x=time_value,y=smoothed_cli,color="Unadjusted")) +
+  geom_line(aes(x=time_value,y=smoothed_adj_cli,color="Adjusted")) +
+  facet_wrap(vars(geo_value)) +
+  labs(x="Time",y="%CLI",title="Doctor visit signal, with and without weekday adjustment, for nine states")
+```
+
+### Evaluating the Indicator
+We can evaluate the Doctor Visits indicator by measuring its correlation with cases. We hope that when the Doctor Visits indicator is high, cases are also high. We can measure this correlation across two axes: space and time Let $c_{it}$ and $d_{it}$ be the confirmed COVID-19 rate and DV indicator respectively, at location $i$ and time $t$. The space-wise correlation of an individual location $i$ is the rank correlation of $c_{\cdot t}$ and $d_{\cdot t}$, and the time-wise correlation of a single day $t$ is the rank correlation of $c_{i \cdot}$ and $d_{i \cdot}$. Below, we plot the distribution of time-wise correlation values for each county with at least 500 total confirmed cases. Most counties have high time-wise correlation values, indicating that when the signal is high in a certain county, the county is likely to have high case incidence.
```

Comment:
Do you want to also show the average time-wise correlation over all counties with cumulative cases > N (or with population > N) as a function of N?  This will show whether the cases of poor correlation are due to small counts (less differentiates power).